<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="verticalFrame">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="ParameterBtn">
           <property name="text">
            <string>图像参数运算</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="EnhanceBtn">
           <property name="text">
            <string>图像增强</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="TransformBtn">
           <property name="text">
            <string>图像变换</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RecoverBtn">
           <property name="text">
            <string>图像恢复</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CodingBtn">
           <property name="text">
            <string>图像编码</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="OpenImage"/>
    <addaction name="SaveImage"/>
    <addaction name="FreshImage"/>
    <addaction name="separator"/>
    <addaction name="ActionExit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="ActionDemo"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="ActionInfo"/>
    <addaction name="Actionhomepage"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="OpenImage">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="SaveImage">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="ActionExit">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="ActionDemo">
   <property name="text">
    <string>实验说明</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>平台信息</string>
   </property>
  </action>
  <action name="ActionInfo">
   <property name="text">
    <string>平台信息</string>
   </property>
  </action>
  <action name="Actionhomepage">
   <property name="text">
    <string>课程主页</string>
   </property>
  </action>
  <action name="FreshImage">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
